production/scrapers/south_dakota_population.R

```diff
@@ -13,8 +13,18 @@ south_dakota_population_check_date <- function(x, date = Sys.Date()){
 }
 
 south_dakota_population_pull <- function(x){
-    get_src_by_attr(x, "a", attr = "href", 
-                    attr_regex = "(?i)documents/AdultPopulation.*.pdf")
+    x <- 'https://doc.sd.gov/about/stats/adult/' %>%
+          read_html() %>%
+          html_nodes('a') %>%
+          html_attr('href') %>%
+          as.data.frame() %>%
+          plyr::rename(c('.' = 'links')) %>%
```

Comment:
Just a noob question for my own knowledge -- what's the `plyr` library? What does this line do for us?

Reply:
Need to avoid the `plyr` library ....it's not maintained and does not play nice with `dplyr`. There's a `tidyverse` equivalent called `rename` (https://dplyr.tidyverse.org/reference/rename.html)